libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.h

```diff
@@ -361,7 +362,7 @@ extern NSString* const kSFRestIfUnmodifiedSince;
  * @param haltOnError Controls whether Salesforce should stop processing subrequests if a subrequest fails.
  * @see https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_batch.htm
  */
-- (SFRestRequest *) batchRequest:(NSArray<SFRestRequest*>*) requests haltOnError:(BOOL) haltOnError;
+- (nonnull SFRestRequest *) batchRequest:(nonnull NSArray<SFRestRequest*>*) requests haltOnError:(BOOL) haltOnError;
```

Comment:
NB: `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END` is employed in this class, so `nonnull` markers technically aren't necessary.  But explicit markers are also helpful in context, so I don't feel strongly about it.